javascript/javascript1/week4/voice.js

```diff
@@ -0,0 +1,105 @@
+let Name1=[];
+let todo=[];
+let a=5;
+let b=3;
+let settimer=4;
+
+
+const days = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"];
+
+getReply("Hello my name is Benjamin")// "Nice to meet you benjamin"
+getReply("What is my name?")  //ur name is Benjamin"
+getReply("Add fishing to my todo")  // "fishing added to your todo"//
+getReply("Add singing to my todo")  // "fishing added to your todo"
+getReply("Add cycling to my todo") 
+getReply("Remove fishing from my todo")
+getReply("What is on my todo")
+getReply("What day is it today")
+getReply("Add(+) a and b value")
+getReply("subract(-) a and b value")
+getReply("multiply(*) a and b value")
+getReply("div(/) a and b value")
+getReply("set timer for 4 mins")
+
+function getReply(cmd)
+{
+    if(cmd.includes("Hello","name"))
+    {
+    Name1 = cmd.slice(16);
+    console.log("Hello" + Name1 + " Nice to meet you!");
+    return 
+    }
+    else if(cmd.includes("my name"))
```

Comment:
the same here, what voice assistant has to answer if user has not mentioned the name yet? in this case you can check if an array with users is not empty or includes that name, and give the right response based on that 😉